src/components/Map/Layers/MapboxLayer.jsx

```diff
@@ -0,0 +1,49 @@
+import { useState, useEffect } from 'react';
+import TileLayer from 'ol/layer/Tile';
+import * as olSource from 'ol/source';
+
+function MapboxLayer({
+  map, zIndex = 1, opacity = 1, styleUrl, accessToken,
+}) {
+  const [mapboxLayer, setMapboxLayer] = useState(false);
```

Comment:
```suggestion
  const [mapboxLayer, setMapboxLayer] = useState();
```

A variable should not store heterogeneous data.
For the absence of data, we should use `undefined` instead.